apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -82,6 +83,19 @@ export const commands = {
     }
   },
 
+  makeEnvironmentSprites(animation, group, bitmap) {
+    const scale = APP_HEIGHT / bitmap.length;
+    for (let i = 0; i < bitmap.length; i++) {
+      for (let j = 0; j < bitmap[0].length; j++) {
+        const location = {
+          x: scale / 2 + scale * i,
```

Comment:
mind adding a comment documenting this calculation?

Reply:
Updated here: https://github.com/code-dot-org/code-dot-org/pull/57107